### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by ifbfirst**
Task: create quiz app
Screenshot: 
image

Deployment: https://ifbfirst-quiz.netlify.app/
Done 11.07.2024 / deadline -
- The project has been initialized using react, vite and TS 
- Сonfigured Prettier, ESlint, Husky
- Add UI for app

### CODE REVIEW

File: src/components/Difficulty.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      <select>
+        <option value="light">light</option>
+        <option value="medium">medium</option>
+        <option value="hard">hard</option>
+      </select>
+    </div>
+  );
+}
+
+export default Difficulty;
```
Comment (EugeneSolovyev) [inline src/components/Difficulty.tsx]: let's make a generic SelectComponent 'cause now there are duplications in Difficulty, Category, Time, and others, however these components have the same structure: div as a wrapper, title and a select
let's think how we can handle that :)

File: src/components/StarsButton.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+function StarsButton() {
+  return <button className="stars-btn">See my stats</button>;
```
Comment (EugeneSolovyev) [inline src/components/StarsButton.tsx]: the same for StarsButton and StartButton

**Review by EugeneSolovyev — COMMENTED** [review]
👍 good
but please check the comments there are a few things to improve tho

File: src/components/Difficulty.tsx (hunk 1 above)
Comment (ifbfirst) [reply src/components/Difficulty.tsx]: no problem, I will menage with it

File: src/components/InputComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+type InputProps = {
+  className: string;
+  text: string;
+};
+
+export const InputComponent = (props: InputProps) => {
+  return (
+    <div className={props.className}>
+      <div>{props.text}</div>
+      <input type="type" />
```
Comment (EugeneSolovyev) [inline src/components/InputComponent.tsx]: please update this component
there is no input with type="type"
looks like you wanted to use it from props
```tsx
export const InputComponent = (props: InputProps) => {
  return (
    <div className={props.className}>
      <div>{props.text}</div>
      <input type={props.type} />
```

**Review by EugeneSolovyev — APPROVED** [review]

**Pull request merged**